packages/core/src/redux/store.ts

```diff
@@ -42,6 +42,22 @@ export const rootReducer = combineReducers({
     waitingParticipants: waitingParticipantsSlice.reducer,
 });
 
+export const rootReducer: AppReducer = (state, action) => {
+    // Reset store state on reset signal
+    if (action.type === APP_RESET_ACTION_TYPE) {
```

Comment:
```suggestion
    if (doAppReset.match(action)) {
```
Each action has a `.match` function, so we can import the action creator and do this instead. No need to export only the type as a string from the slice either then.

https://redux-toolkit.js.org/api/createAction#actioncreatormatch